app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,50 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_user!
+      before_action :check_book_availability, only: [:create]
+      before_action :clean_pagination
+      respond_to :json
+
+      def index
+        rents = Rent.where(user_id: current_user.id).page(@page).per(@page_size)
+        render json: rents, each_serializer: Api::V1::RentSerializer
+      end
+
+      def create
+        rent = Rent.new(rent_params) do |r|
+          r.user_id = current_user.id
+        end
+        render json: rent, serializer: Api::V1::RentSerializer and return if rent.save
+
+        render json: rent.errors and return
+      end
+
+      def check_book_availability
+        book = Book.find_by(id: rent_params[:book_id])
+        rent_error(book) and return if !book || !book.available?
+      end
+
+      def rent_error(rent)
+        if !rent
```

Comment:
try to avoid the use of `!` and `if..elsif..end`. Use guard clauses instead:
`return render json: {blabla}, status: not_found if rent.blank?`
`return render json: {} status: unprocessable_entity unless rent.available?`
`render json: {error: 'whatever you render if the execution reaches this line'}`